webapp/playbooks/src/components/sidebar/sidebar.tsx

```diff
@@ -51,11 +51,17 @@ const Sidebar = (props: SidebarProps) => {
                     delay={OVERLAY_DELAY}
                     shouldUpdatePosition={true}
                     overlay={
-                        <Tooltip id='team-name__tooltip'>{team?.description?.length ? team.description : 'No team is selected'}</Tooltip>
+                        <Tooltip id='team-name__tooltip'>
+                            {
+                                team?.description?.length ? team.description : 'No team is selected' // eslint-disable-line formatjs/no-literal-string-in-jsx
```

Comment:
Sweet, `formatjs/no-literal-string-in-jsx` must handle string literals in ternaries now. 

Rather than commenting/ignoring out all of them, let's make these translatable if they should be.
```suggestion
                                team?.description?.length ? team.description : formatMessage({defaultMessage: 'No team is selected'})
```

Reply:
added the translations